docs/upgrade.html

```diff
@@ -66,12 +66,22 @@
     <li> The new Java Consumer now allows users to search offsets by timestamp on partitions. </li>
     <li> When using an Authorizer and a user hasn't got <b>Describe</b> authorization on a topic, the broker will no longer return TOPIC_AUTHORIZATION_FAILED errors  
          but just UNKNOWN_TOPIC_OR_PARTITION errors, to avoid leaking topic names.</li>
+    <li> Fetch responses have a size limit by default (50 MB for consumers and 10 MB for replication). Note that this is not an absolute maximum as explained in the
```

Comment:
Is it worth pointing out that we previously only had per-partition fetch limits?